README.md

```diff
@@ -1,5 +1,16 @@
 # Open Sesame: an Open Source Mentorship Platform
 Open Sesame is a platform intended to make contributing to open source less intimidating by matching first-time contributors to experienced mentors.
 
+## React Integration
+This platform makes use of the [React framework](https://reactjs.org/). In order to provide React functionality to the projects section while allowing for non-React HTML and JS
```

Comment:
Nit. "Projects section" may be a bit vague. Maybe "project browsing section" or "project search page."

Reply:
Agreed. Changing.